src/content/blog/ddev-bundled-tools-using-custom-versions.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "How-to Downgrade Terminus in DDEV's Web Container and Customize Other Bundled Tools"
```

Comment:
```suggestion
title: "How to Downgrade Terminus in DDEV's Web Container and Customize Other Bundled Tools"
```

Felt strange before. Reads better as a sentence, or "A How-to to Downgrade ..." , or "How-to: Downgrade ..."